app/views/candidate_interface/gcse/science/grade/awards_edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :title, title_with_error_prefix(t('gcse_edit_grade.page_title', subject: @subject), @gcse_grade_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @gcse_grade_form, url: candidate_interface_gcse_science_edit_grade_path, method: :patch do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl"><%= t('gcse_edit_grade.page_title', subject: @subject) %></h1>
+
+      <%= render CandidateInterface::GcseGradeGuidanceComponent.new(@subject, @qualification_type) %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
There needn’t be any grid layout on this part of the page, so this can come out. Currently this reduced the width of the form.

Reply:
👍